src/styles/blocks/clock.scss

```diff
@@ -0,0 +1,44 @@
+.clock {
+  position: relative;
+  @include square($clockSize);
+  border: 3px dotted black;
+  border-radius: 50%;
+  justify-content: center;
+  align-items: center;
+
+  &::after {
+    content: "";
+    position: absolute;
+    top: 50%;
+    left: 50%;
```

Comment:
If you use a number / color several times, it is better to move it to a variable. This also applies to all subsequent similar cases